### ISSUE TITLE
werid learning rate occur

### ISSUE THREAD

**Issue by congjianting** [facebookresearch/ConvNeXt#7]
HI,

i trained convnext model in my project, but i found werid lr happened.

i set lr = 0.002, but in the training loop, the lr param seems to be only 0.00002.  

someone knowns why? 

![convnext_1](https://user-images.githubusercontent.com/20871348/149315637-137e7faf-cb22-44a2-b217-545a4533d4c7.jpeg)
![3661642070433_ pic](https://user-images.githubusercontent.com/20871348/149315655-c7cca138-54f1-42a0-b8ca-5bdcbf9fb5fd.jpg)

**Comment by congjianting** [comment]
the trainning loop image likes the following, 

![convnext_2](https://user-images.githubusercontent.com/20871348/149316220-040c4b47-64e3-47ad-b69c-5deacfb5f09d.jpeg)

**Comment by congjianting** [comment]
i have 8 GPU and 1 image per GPU, so the batchsize=8

**Comment by s9xie** [comment]
Hi @congjianting,

This is because of the use of the layer decay. In mmcv the logger by default will log the minimum lr.
https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/logger/text.py#L216

We just pushed a fix and now a customized logger will output both `lr` and `layer_0_lr`, representing the minimum lr and maximum lr (default lr) used in the network. 

Thank you for reporting this logging issue and feel free to reopen the case if you have more questions.
